spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/FPN.scala

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright 2016 The BigDL Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.intel.analytics.bigdl.nn
+
+import com.intel.analytics.bigdl.nn.Graph.ModuleNode
+import com.intel.analytics.bigdl.nn.abstractnn.AbstractModule
+import com.intel.analytics.bigdl.tensor.Tensor
+import com.intel.analytics.bigdl.tensor.TensorNumericMath.TensorNumeric
+import com.intel.analytics.bigdl.utils.Table
+
+import scala.reflect.ClassTag
+
+/**
+ * Feature Pyramid Network.
+ * @param in_channels_list number of channels of feature maps
+ * @param out_channels number of channels of FPN output
+ */
+
+class FPN[T : ClassTag](
+  val in_channels_list: Array[Int],
+  val out_channels: Int
+)
+  (implicit ev: TensorNumeric[T])
+  extends AbstractModule[Table, Table, T]{
```

Comment:
this class can extend `BaseModule` instead of `AbstractModule`

Reply:
Changed